fedbiomed/common/_config.py

```diff
@@ -0,0 +1,246 @@
+# This file is originally part of Fed-BioMed
+# SPDX-License-Identifier: Apache-2.0
+
+import configparser
```

Comment:
Sorry if I'm missing something, but I don't really understand what is this `_config` for.
- most of the code seems to duplicate `config.py`
- I could not find any use of it (`'config.py` seems to be used)

Same for `fedbiomed/node/_config.py`